### PR TITLE
Argument/command name parsing fix

### PR DESCRIPTION
**Pull request by samsamson33**

### CODE REVIEW

File: events/message.js [hunk 1]
```diff
@@ -18,18 +18,20 @@ module.exports = async (Discord, client, message) => {
...
-	if (permission <= 1 && message.content.toLowerCase().startsWith("suggester:")) prefix = "suggester:";
-	if (permission <= 1 && message.content.toLowerCase().startsWith(`${client.user.id}:`)) prefix = `${client.user.id}:`;
+	if (permission <= 1 && message.content.toLowerCase().startsWith("suggester:")) serverPrefix = "suggester:";
+	if (permission <= 1 && message.content.toLowerCase().startsWith(`${client.user.id}:`)) serverPrefix = `${client.user.id}:`;
 	if (!message.content.toLowerCase().startsWith(serverPrefix)) return;
 	//Only commands after this point
 	//Check if message is a command
 	fs.readdir("./commands/", (err, files) => {
+		//Input command
+		const commandText = message.content.slice(serverPrefix.length).split(" ")[0].toLowerCase();
```
Comment (BrightnessTM) [inline events/message.js]: This doesn't seem to work as expected when I test it, but we probably shouldn't be defining commandText a bunch of times so moving it outside the `forEach` should be fine. Maybe just use the old code for this
```suggestion
		const commandText = message.content.split(" ")[0].toLowerCase().split(serverPrefix)[1];
```

File: events/message.js [hunk 2]
```diff
@@ -18,18 +18,20 @@ module.exports = async (Discord, client, message) => {
 	let possiblementions = [`<@${client.user.id}> help`, `<@${client.user.id}>help`, `<@!${client.user.id}> help`, `<@!${client.user.id}>help`, `<@${client.user.id}> prefix`, `<@${client.user.id}>prefix`, `<@!${client.user.id}> prefix`, `<@!${client.user.id}>prefix`, `<@${client.user.id}> ping`, `<@${client.user.id}>ping`, `<@!${client.user.id}> ping`, `<@!${client.user.id}>ping`];
 	if (possiblementions.includes(message.content.toLowerCase())) return message.reply(`Hi there! My prefix in this server is ${Discord.escapeMarkdown(serverPrefix)}\nYou can read more about my commands at https://suggester.gitbook.io/`);
 
-	if (permission <= 1 && message.content.toLowerCase().startsWith("suggester:")) prefix = "suggester:";
-	if (permission <= 1 && message.content.toLowerCase().startsWith(`${client.user.id}:`)) prefix = `${client.user.id}:`;
+	if (permission <= 1 && message.content.toLowerCase().startsWith("suggester:")) serverPrefix = "suggester:";
```
Comment (BrightnessTM) [inline events/message.js]: This was already added in 792b366

**Review by BrightnessTM — CHANGES_REQUESTED** [review]

File: events/message.js (hunk 1 above)
Comment (samsamson33) [reply events/message.js]: How isn't it working as expected?

File: events/message.js (hunk 2 above)
Comment (samsamson33) [reply events/message.js]: Oops, forgot to merge from upstream. I removed that commit in the force push.

File: events/message.js (hunk 1 above)
Comment (BrightnessTM) [reply events/message.js]: It returns `<prefix>command` instead of `command`.

**Comment by BrightnessTM** [comment]
We're going to be rewriting the command handler, so I'm going to close this issue.

**Pull request closed without merging**